terraform/modules/bosh_vpc/network_private.tf

```diff
@@ -48,31 +47,58 @@ resource "aws_route_table" "az2_private_route_table" {
   }
 }
 
+resource "aws_route_table_association" "az1_private_rta" {
+  subnet_id = "${aws_subnet.az1_private.id}"
+  route_table_id = "${aws_route_table.az1_private_route_table.id}"
+}
+
+resource "aws_route_table_association" "az2_private_rta" {
+  subnet_id = "${aws_subnet.az2_private.id}"
+  route_table_id = "${aws_route_table.az2_private_route_table.id}"
```

Comment:
When we roll this out to staging we should verify that there's no disruption in service doing the route table change in the same TF update as creating the new Gateways.  I'm a little worried that the route table change will be applied before the new VMs are fully booted and forwarding packets.